mks/service/config.py

```diff
@@ -29,6 +29,10 @@ def get_jwt_key():
     return jwk.JWK.from_json('{"k":"%s","kty":"oct"}' % (key, ))
 
 
+def get_raw_key():
+    return os.getenv("RAW_ACCESS_KEY")
```

Comment:
wat als `RAW_ACCESS_KEY` een lege string is? Of None?

Is dat een configuratie error, of moet dat kunnen?

Reply:
Als het een lege string is, is een config error.